tests/lib/Repository/Decorator/NotificationServiceDecoratorTest.php

```diff
@@ -43,6 +44,20 @@ public function testLoadNotificationsDecorator()
         $decoratedService->loadNotifications(...$parameters);
     }
 
+    public function testFindNotificationsDecorator(): void
+    {
+        $serviceMock = $this->createServiceMock();
+        $decoratedService = $this->createDecorator($serviceMock);
+        $typeCriterion = new Type('Workflow:Review');
+        $query = new NotificationQuery([$typeCriterion], 264, 959);
+
+        $serviceMock->expects($this->once())
```

Comment:
AFAIR this one is static too
```suggestion
        $serviceMock->expects(self::once())
```